.golangci.yml

```diff
@@ -1,3 +1,20 @@
+# Copyright 2019 ChainSafe Systems (ON) Corp.
+# This file is part of gossamer.
+#
+# The gossamer library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# The gossamer library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+#
+
```

Comment:
The tool seemed to add an extra line to the license, just a blank comment line. This doesn't exist in the other files. 

Not a big deal, but perhaps we should change it to be consistent?

Reply:
@amerameen done

Reply:
Awesome!